http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/HttpVersionAlpnWithConfigKeyIT.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.ts.http.restclient.reactive;
+
+import static io.quarkus.ts.http.restclient.reactive.resources.HttpVersionClientResource.NAME;
+import static io.quarkus.ts.http.restclient.reactive.resources.HttpVersionClientResource.WRONG_HTTP_VERSION;
+import static org.hamcrest.Matchers.containsString;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class HttpVersionAlpnWithConfigKeyIT {
+
+    String SUCESSFUL_RESPONSE_STRING = "Hello " + NAME;
```

Comment:
Typo in  the word should be SUCCESSFUL_, you could change it in all SUCESSFUL_RESPONSE_STRING